core/scm.js

```diff
@@ -192,6 +192,10 @@ const SCHEMA_HOOK = Joi.object().keys({
 
     username: Joi.reach(SCHEMA_USER, 'username'),
 
+    commitAuthors: Joi.array()
```

Comment:
I think this should be `Joi.array().items(Joi.string())`

Reply:
change done, is there something else that I should change?